VMKExample/VMKExample.xcodeproj/project.pbxproj

```diff
@@ -600,7 +606,7 @@
 				DEFINES_MODULE = YES;
 				DEVELOPMENT_TEAM = 85XT4HAH76;
 				INFOPLIST_FILE = VMKExample/Info.plist;
-				IPHONEOS_DEPLOYMENT_TARGET = 9.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 12.0;
```

Comment:
The pr name is "bumo to iOS 15", but it has upgraded to 12, its confusing. what is the reason for this?

Reply:
branch name refers to the epic which is bump PRO to support iOS15. 
For VMK it's not necessary, so i put version 12... but you're right, it can also be version 15